pulp-pypi-sync-job/base/cronjob.yaml

```
@@ -0,0 +1,92 @@
+---
+kind: CronJob
+apiVersion: batch/v1beta1
+metadata:
+  name: pulp-pypi-sync-job
+  labels:
+    app: thoth
+    component: pulp-pypi-sync-job
+spec:
+  schedule: "* */6 * * *"
+  suspend: true
+  concurrencyPolicy: Forbid
+  successfulJobsHistoryLimit: 2
+  failedJobsHistoryLimit: 4
```

```suggestion
  successfulJobsHistoryLimit: 1
  failedJobsHistoryLimit: 1
```

let's keep this reduced, only failed pod is good enough for us.wdyt?